packages/perseus/src/widgets/number-line/number-line.test.ts

```diff
@@ -183,41 +188,91 @@ describe("number-line widget", () => {
         });
     });
 
-    it("can be answered correctly", () => {
-        // Arrange
-        const apiOptions: APIOptions = {
-            isMobile: false,
-        };
-        const {renderer} = renderQuestion(question1, apiOptions);
-
-        // Act
-        const [numberLine] = renderer.findWidgets("number-line 1");
-        act(() => numberLine.movePosition(-2.5));
-        const score = scorePerseusItemTesting(
-            question1,
-            renderer.getUserInputMap(),
-        );
-
-        // assert
-        expect(score).toHaveBeenAnsweredCorrectly();
+    const numberLineOptions: PerseusNumberLineWidgetOptions = {
+        labelRange: [null, null],
+        initialX: null,
+        tickStep: 1,
+        labelStyle: "decimal",
+        labelTicks: true,
+        isInequality: false,
+        snapDivisions: 2,
+        range: [-4, 4],
+        static: false,
+        correctRel: "eq",
+        numDivisions: null,
+        divisionRange: [1, 10],
+        correctX: -2.5,
+    };
+
+    test("answerless data doesn't contain answers", () => {
+        const options = getAnswerlessItem("number-line", numberLineOptions)
+            .question.widgets["number-line 1"].options;
+        expect(options.correctX).toBeUndefined();
+        expect(options.correctRel).toBeUndefined();
     });
 
-    it("can be answered incorrectly", () => {
-        // Arrange
-        const apiOptions: APIOptions = {
-            isMobile: false,
-        };
-        const {renderer} = renderQuestion(question1, apiOptions);
-
-        // Act
-        const [numberLine] = renderer.findWidgets("number-line 1");
-        act(() => numberLine.movePosition(3.5));
-        const score = scorePerseusItemTesting(
-            question1,
-            renderer.getUserInputMap(),
-        );
+    describe.each([
+        ["answerless", getAnswerlessItem("number-line", numberLineOptions)],
+        ["answerful", getAnswerfulItem("number-line", numberLineOptions)],
+    ])("given %s options", (_, {question}) => {
+        const answerKey = getAnswerfulItem(
+            "number-line",
+            numberLineOptions,
+        ).question;
+
+        it("can be answered correctly", () => {
+            // Arrange
+            const apiOptions: APIOptions = {
+                isMobile: false,
+            };
```

Comment:
`isMobile` defaults to `false`, so I don't think you need to define your own APIOptions for these tests.